testing/integration-neo4j/src/test/java/com/neueda/jetbrains/plugin/graphdb/test/integration/neo4j/tests/cypher/inspection/CypherExplainWarningInspectionTest.java

```diff
@@ -0,0 +1,60 @@
+package com.neueda.jetbrains.plugin.graphdb.test.integration.neo4j.tests.cypher.inspection;
+
+import com.intellij.psi.PsiFile;
+import com.neueda.jetbrains.plugin.graphdb.jetbrains.component.datasource.state.DataSourceApi;
+import com.neueda.jetbrains.plugin.graphdb.jetbrains.inspection.CypherExplainWarningInspection;
+import com.neueda.jetbrains.plugin.graphdb.jetbrains.util.NameUtil;
+import com.neueda.jetbrains.plugin.graphdb.test.integration.neo4j.util.base.BaseIntegrationTest;
+
+public class CypherExplainWarningInspectionTest extends BaseIntegrationTest {
```

Comment:
We are go going to add more inspections in future. Let's extract some of the logic in "BaseInspectionTest"

Logic to extract:
- Inspection class to enable
- Configure from existing existing file + check highligting
- Create data source file + add file to project